ui-v2/app/styles/components/list-collection.scss

```diff
@@ -13,15 +13,27 @@
 .unhealthy > div {
   margin-bottom: 20px;
 }
+.healthy > div > ul > li {
+  padding-right: 23px;
```

Comment:
Not a big deal at all, but for your reference, I've tried making all spacing stick to the 12, 24, 36, 48, etc., as suggested by [Structure](https://share.goabstract.com/68ac9203-2c5f-4033-9521-e932483569c3), so I was surprised to see `23`, but again, not a big deal, and we can go back and do a spacing audit another time. 

![image](https://user-images.githubusercontent.com/6323933/46553904-5de0b780-c8a4-11e8-9452-a63f58a2209c.png)

Reply:
Hey!

Cool yeah for sure. These are only `+`s because I moved them from further below, the values haven't been changed since my initial trace of the original layouts back from when we did the initial release, which would have been before Structure.

At some point I'll be able to do a re-trace of everything, hopefully might get a chance in the next couple of weeks.

Ta

John